selfdrive/ui/translations/main_ko.ts

```diff
@@ -1004,15 +1004,15 @@ location set</source>
     </message>
     <message>
         <source>Experimental mode</source>
-        <translation type="unfinished"></translation>
+        <translation>실험 모드</translation>
     </message>
     <message>
         <source>&lt;b&gt;WARNING: openpilot longitudinal control is experimental for this car and will disable AEB.&lt;/b&gt;&lt;br&gt;          openpilot defaults to the car&apos;s built-in ACC instead of openpilot&apos;s longitudinal control on this car. Enable this to switch to openpilot longitudinal control.</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;경고:  openpilot 롱컨트롤은 실험적인 기능으로 차량의 AEB를 비활성화합니다.&lt;/b&gt;&lt;br&gt;          openpilot은 차량의 내장 ACC로 기본 설정됩니다. 롱컨트롤으로 전환하려면 이 옵션을 활성화하세요.</translation>
     </message>
     <message>
         <source>    openpilot defaults to driving in &lt;b&gt;chill mode&lt;/b&gt;.    Experimental mode enables &lt;b&gt;alpha-level features&lt;/b&gt; that aren&apos;t ready for chill mode.     Experimental features are listed below:    &lt;br&gt;     &lt;h4&gt;🌮 End-to-End Longitudinal Control 🌮&lt;/h4&gt;     Let the driving model control the gas and brakes. openpilot will drive as it thinks a human would, including stopping for red lights and stop signs.</source>
-        <translation type="unfinished"></translation>
+        <translation>    openpilot은 &lt;b&gt;chill 모드&lt;/b&gt;로 기본 설정됩니다.    실험 모드는 chill 모드에 준비되지 않은 &lt;b&gt;알파 수준 기능&lt;/b&gt;을 활성화 합니다.     실험 모드의 특징은 아래에 나열되어 있습니다    &lt;br&gt;     &lt;h4&gt;🌮 E2E 롱컨트롤 🌮&lt;/h4&gt;     주행모델이 가속과 감속을 제어하도록 합니다. openpilot은 신호등과 정지표지판을 보고 멈추는 것을 포함하여 운전자가 생각하는것처럼 주행합니다.      </translation>
```

Comment:
> no translation for chill? or not needed?

Could not find a suitable word to express chill. Chill Mode / Cool(쿨) Mode /  Relax(여유) Mode / Comfort(편안함) Mode

I thought about many things and talked to them in the group chat room, but I applied it as it was because I thought it would be better to stay in chill mode